### ISSUE TITLE
fridge temperatures monitoring

### ISSUE THREAD

**Issue by amunizp** [Basicprogrammer10/WaterTemp#10]
Hello!
Thanks for this great project, found it on featured on instructables!

Is there a way to add more thermocouples? In food making locations one needs to monitor all fridge temperatures and this logger seems to fit the bill!

Up to now I used brewpi and I can add as many RJ connected thermocouples as I like as it seems to detect it as a separate addresses.

I do not need the complexity of brewpi and I think this project could do the trick!

**Comment by connorslade** [comment]
Thanks for finding interest in my project :P

With the one Wire protocol, you can add as many sensors as you want in parallel. However, currently my software only supports one sensor, but allowing it to work with more than one wouldn't be too difficult. The only problem would be displaying all the data on the web interface. 

For your use case, would you need to see all the temperatures separately and maybe get alerts if the temperature goes out of a set range. Or just to log the data form the different sensors to get from the data download page / Backend API?

**Comment by amunizp** [comment]
Alerts on temperature reaching the danger zone would be amazing. Currently I look into it from time to time. Logging is needed. https://www.food.gov.uk/safety-hygiene/chilling

**Comment by connorslade** [comment]
Ok, I'm working on adding support for more than one sensor now. 
I can have it send emails / webhooks if the temperature goes in that danger zone, and I think I will add a new page (through a plugin) to show the temperatures of all the thermometers.

**Comment by connorslade** [comment]
The backend code for the Raspberry Pi now supports multiple sensors : ) ([In the multiSensor branch](https://github.com/Basicprogrammer10/WaterTemp/tree/multiSensor))
I just need to test it with the real sensors (done :P) and make the web server plugin (Done)

![Screen Shot 2021-08-03 at 05 55 00](https://user-images.githubusercontent.com/50306817/127965135-98f83734-dde4-4762-8e31-3ca8b223a2a7.png)

**Comment by connorslade** [comment]
I think I got everything working (Multi sensors, Web interface, Alerts). 

So if you want to try it (and give some feedback) that would be great.

It's all in the multiSensor branch. 

To support more than one sensor the config file on the sensor interface was updated so each sensor gets one line that looks like this:

```
sensor_[FriendlyName] = [SensorId], [CalibrationFactor]
```

So for example my sensor config looks like this:

```
sensor_water = 28-00000bdf4372, 0.0
```

And now for the web server there is a new config file for alerts at `api/plugins/multiSensor/config.js`

The comments there explain everything fairly well. 

It is really customizable, you can add multiple alerts with different types that run for certain (or all) sensors. For sending the alerts it can now send emails and discord web hooks.

**Comment by connorslade** [comment]
Multi Sensor branch has been merged into Master :P
